deploy/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -376,23 +376,16 @@ rules:
   - get
   - list
 ---
+# TODO: remove this, once https://github.com/rook/rook/issues/10141
+# is resolved.
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: cephfs-csi-nodeplugin
 rules:
   - apiGroups: [""]
-    resources: ["namespaces"]
-    verbs: ["get", "list"]
-  - apiGroups: [""]
-    resources: ["persistentvolumes"]
-    verbs: ["get", "list", "watch", "update"]
-  - apiGroups: ["storage.k8s.io"]
-    resources: ["volumeattachments"]
-    verbs: ["get", "list", "watch", "update"]
-  - apiGroups: [""]
-    resources: ["configmaps"]
-    verbs: ["get", "list"]
+    resources: ["nodes"]
+    verbs: ["get"]
```

Comment:
Usually, when I have needed "get" permissions, I have also needed "list" permissions. Is "get" really all that is necessary?

Reply:
We really dont need this also, this is a workaround for https://github.com/rook/rook/issues/10141 as we did for nfs https://github.com/rook/rook/pull/10183, even we can have empty rules for this one also